src/main/scala/cromwell/binding/IOInterface.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.binding
```

Comment:
When I was looking at it earlier I was hung up on that abstract type bit we discussed in person. I think you're right here.